# Welcome to MMmicrobiome Lab

We are two research groups centered on exploring the dynamics of bacterial genomes, bacterial functions, and host-microbe interactions in the context of health versus disease and throughout human evolution.

Our primary objective is to gain a functional understanding of host-microbe systems. We specifically concentrate on assembling and characterizing an extensive collection of human gut microbiomes and bacterial strains in pure culture from diverse global populations. To achieve this, we employ in silico, in vitro, and in vivo approaches to identify actionable features within host-associated microbiomes that can contribute to enhancing human health.


## Key Research Areas

Our key research areas are:

- **Global Microbiome Conservancy:** Professors Poyet and Groussin established the Global Microbiome Conservancy (GMbC) in 2016, an international non-profit initiative dedicated to advancing microbiome science equitably and ethically.
- **Synthetic Bacterial Communities as Therapeutics for Gastrointestinal Disorders:** We employ a rational design approach to develop microbiota-based intervention strategies aimed at modulating the diversity, composition, and function of perturbed microbiomes, particularly for gastrointestinal disorders.
- **Evolution of IgA-Bacteria Interactions:** Our research explores the association between host lifestyle, microbiome compositions, and interactions between IgA antibodies and bacteria within the human metaorganism.
- **Anaerobic Culturing:** We focus on isolating and characterizing bacterial strains associated with humans, obtained from stool samples and intestinal biopsies, with a particular emphasis on anaerobic culturing.
- **High Throughput Identification of Antigens/Peptides Associated with Autoimmune Diseases:** Implementing innovative techniques such as Phage Immunoprecipitation Sequencing (PhIP-Seq) and a Yeast Surface Display platform, we aim to identify antigens and peptides linked to autoimmune diseases.
- **Multiomics Investigations of Microbiomes:** Our research involves employing multiomics approaches to comprehensively study microbiomes, providing a holistic understanding of their compositions and functions.
- **Origin, Evolution, and Mechanism of Cholesterol Metabolism by the Human Gut Microbiome:** Exploring a novel approach, we investigate ways to modulate the gut microbiome to reduce blood cholesterol levels through microbial degradation of intestinal cholesterol.
- **Primary Sclerosing Cholangitis and the Gut Microbiome:** Our goal is to identify biological markers and explore the contribution of the gut-liver axis to the pathogenesis of Primary Sclerosing Cholangitis (PSC).

## Newcomer

If you are a new comer, make sure you go through our [new comer](setup)(ADD link to onboarding).

## Coding guidelines

To be done.


## Publications
1. Zhao, S.; Lieberman, T.D.; Poyet, M.; Kauffman, K.M.; Gibbons, S.M.; Groussin, M.; Xavier, R.J.;
Alm, E.J. Adaptive Evolution within Gut Microbiomes of Healthy People. Cell Host Microbe 2019,
25, 656-667.e8, [doi:10.1016/j.chom.2019.03.007](https://www.sciencedirect.com/science/article/pii/S1931312819301593) .
2. Schaan, A.P.; Vidal, A.; Zhang, A.-N.; Poyet, M.; Alm, E.J.; Groussin, M.; Ribeiro-dos-Santos, Â.
Temporal Dynamics of Gut Microbiomes in Non-Industrialized Urban Amazonia; Microbiology,
2023;[doi:10.1101/2023.05.11.540371](https://www.biorxiv.org/content/10.1101/2023.05.11.540371v1)
3. Rühlemann, M.C.; Bang, C.; Gogarten, J.F.; Hermes, B.M.; Groussin, M.; Waschina, S.; Poyet, M.;
Ulrich, M.; Akoua-Koffi, C.; Deschner, T.; et al. Comparative Metagenomics Reveals Host-Specific
Functional Adaptation of Intestinal Microbiota across Hominids 2023, [doi:10.1101/2023.03.01.530589](https://www.biorxiv.org/content/10.1101/2023.03.01.530589v1.full.pdf); 
4. Poyet, M.; Roux, V.L.; Gibert, P.; Meirland, A.; Prévost, G.; Eslin, P.; Chabrerie, O. The Wide
Potential Trophic Niche of the Asiatic Fruit Fly Drosophila Suzukii: The Key of Its Invasion Success in
Temperate Europe? PLOS ONE 2015, 10, e0142785, [doi:10.1371/journal.pone.0142785.](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0142785)
5. Poyet, M.; Havard, S.; Prevost, G.; Chabrerie, O.; Doury, G.; Gibert, P.; Eslin, P. Resistance of
Drosophila Suzukii to the Larval Parasitoids Leptopilina Heterotoma and Asobara Japonica Is
Related to Haemocyte Load. Physiol. Entomol. 2013, 38, 45–53, [doi:10.1111/phen.12002](https://resjournals.onlinelibrary.wiley.com/doi/10.1111/phen.12002).
6. Poyet, M.; Eslin, P.; Héraude, M.; Le Roux, V.; Prévost, G.; Gibert, P.; Chabrerie, O. Invasive Host for
Invasive Pest: When the Asiatic Cherry Fly (Drosophila Suzukii) Meets the American Black Cherry
(Prunus Serotina) in Europe. Agric. For. Entomol. 2014, 16, 251–259, [doi:10.1111/afe.12052.](https://resjournals.onlinelibrary.wiley.com/doi/full/10.1111/afe.12052)
7. Poyet, M.; Groussin, M.; Gibbons, S.M.; Avila-Pacheco, J.; Jiang, X.; Kearney, S.M.; Perrotta, A.R.;
Berdy, B.; Zhao, S.; Lieberman, T.D.; et al. A Library of Human Gut Bacterial Isolates Paired with
Longitudinal Multiomics Data Enables Mechanistic Microbiome Research. Nat. Med. 2019, 25,
1442–1452, [doi:10.1038/s41591-019-0559-3.](https://www.nature.com/articles/s41591-019-0559-3)
8. Poyet, M.; Eslin, P.; Chabrerie, O.; Prud’homme, S.M.; Desouhant, E.; Gibert, P. The Invasive Pest
Drosophila Suzukii Uses Trans-Generational Medication to Resist Parasitoid Attack. Sci. Rep. 2017,
7, 43696, [doi:10.1038/srep43696.](https://www.nature.com/articles/srep43696)
9. Mazel, F.; Davis, K.M.; Loudon, A.; Kwong, W.K.; Groussin, M.; Parfrey, L.W. Is Host Filtering the
Main Driver of Phylosymbiosis across the Tree of Life? mSystems 2018, 3,
10.1128/msystems.00097-18, [doi:10.1128/msystems.00097-18.](https://journals.asm.org/doi/10.1128/msystems.00097-18)
10. Kearney, S.M.; Gibbons, S.M.; Poyet, M.; Gurry, T.; Bullock, K.; Allegretti, J.R.; Clish, C.B.; Alm, E.J.
Endospores and Other Lysis-Resistant Bacteria Comprise a Widely Shared Core Community within
the Human Microbiota. ISME J. 2018, 12, 2403–2416, [doi:10.1038/s41396-018-0192-z.](https://www.nature.com/articles/s41396-018-0192-z)
11. Jiang, X.; Hall, A.B.; Arthur, T.D.; Plichta, D.R.; Covington, C.T.; Poyet, M.; Crothers, J.; Moses, P.L.;
Tolonen, A.C.; Vlamakis, H.; et al. Invertible Promoters Mediate Bacterial Phase Variation,
Antibiotic Resistance, and Host Adaptation in the Gut. Science 2019, 363, 181–187,
[doi:10.1126/science.aau5238.](https://www.science.org/doi/10.1126/science.aau5238)
12. Groussin, M.; Poyet, M.; Sistiaga, A.; Kearney, S.M.; Moniz, K.; Noel, M.; Hooker, J.; Gibbons, S.M.;
Segurel, L.; Froment, A.; et al. Elevated Rates of Horizontal Gene Transfer in the Industrialized
Human Microbiome. Cell 2021, 184, 2053-2067.e18, [doi:10.1016/j.cell.2021.02.052.](https://www.sciencedirect.com/science/article/pii/S0092867421002415)
13. Groussin, M.; Mazel, F.; Sanders, J.G.; Smillie, C.S.; Lavergne, S.; Thuiller, W.; Alm, E.J. Unraveling
the Processes Shaping Mammalian Gut Microbiomes over Evolutionary Time. Nat. Commun. 2017,
8, 14319, [doi:10.1038/ncomms14319.](https://www.nature.com/articles/ncomms14319)
14. Groussin, M.; Mazel, F.; Alm, E.J. Co-Evolution and Co-Speciation of Host-Gut Bacteria Systems. Cell
Host Microbe 2020, 28, 12–22, [doi:10.1016/j.chom.2020.06.013.](https://www.sciencedirect.com/science/article/pii/S1931312820303541)
15. Groussin, M.; Hobbs, J.K.; Szöllősi, G.J.; Gribaldo, S.; Arcus, V.L.; Gouy, M. Toward More Accurate
Ancestral Protein Genotype–Phenotype Reconstructions with the Use of Species Tree-Aware Gene
Trees. Mol. Biol. Evol. 2015, 32, 13–22, [doi:10.1093/molbev/msu305.](https://academic.oup.com/mbe/article/32/1/13/2925600)
16. Groussin, M.; Gouy, M. Adaptation to Environmental Temperature Is a Major Determinant of
Molecular Evolutionary Rates in Archaea. Mol. Biol. Evol. 2011, 28, 2661–2674,
[doi:10.1093/molbev/msr098.](https://academic.oup.com/mbe/article/28/9/2661/1013956)
17. Groussin, M.; Boussau, B.; Szöllõsi, G.; Eme, L.; Gouy, M.; Brochier-Armanet, C.; Daubin, V. Gene
Acquisitions from Bacteria at the Origins of Major Archaeal Clades Are Vastly Overestimated. Mol.
Biol. Evol. 2016, 33, 305–310, [doi:10.1093/molbev/msv249.](https://academic.oup.com/mbe/article/33/2/305/2579643)
18. Groussin, M.; Boussau, B.; Gouy, M. A Branch-Heterogeneous Model of Protein Evolution for
Efficient Inference of Ancestral Sequences. Syst. Biol. 2013, 62, 523–538,
[doi:10.1093/sysbio/syt016.](https://academic.oup.com/sysbio/article/62/4/523/1611505)
19. Bousbaine, D.; Fisch, L.I.; London, M.; Bhagchandani, P.; Rezende de Castro, T.B.; Mimee, M.;
Olesen, S.; Reis, B.S.; VanInsberghe, D.; Bortolatto, J.; et al. A Conserved Bacteroidetes Antigen
Induces Anti-Inflammatory Intestinal T Lymphocytes. Science 2022, 377, 660–666,
[doi:10.1126/science.abg5645.](https://www.science.org/doi/10.1126/science.abg5645)


